CSharp/ArticleProjects/TaskDependencies/Program.cs

```diff
@@ -58,105 +56,188 @@ public static void Main(string[] args)
-                    new CloudTask("4", "cmd.exe /c echo 4")
+                        // Tasks 1, 2, and 3 don't depend on any other tasks. Because
+                        // we will be using them for a task range dependency, we must
+                        // specify string representations of integers as their ids.
+                        new CloudTask("1", "cmd.exe /c echo 1"),
+                        new CloudTask("2", "cmd.exe /c echo 2"),
+                        new CloudTask("3", "cmd.exe /c echo 3"),
+ 
+                        // Task 4 depends on a range of tasks, 1 through 3
+                        new CloudTask("4", "cmd.exe /c echo 4")
+                        {
+                            // To use a range of tasks, their ids must be integer values.
+                            // Note that we pass integers as parameters to TaskIdRange,
+                            // but their ids (above) are string representations of the ids.
+                            DependsOn = TaskDependencies.OnIdRange(1, 3)
+                        },
+
+                        // Task 5 depends on a range of tasks, 1 through 3, and 'Flowers'
+                        new CloudTask("5", "cmd.exe /c echo 5")
+                        {
+                            DependsOn = new TaskDependencies(
+                                new[] { "Flowers" },
+                                new[] { new TaskIdRange(1, 3) })
+                        },
+
+                        // Task A is a parent task that exits with a non-zero exit code.
+                        new CloudTask("A", "cmd.exe /c echo A exit 1"),
+                        // Task B depends on task A
+                        new CloudTask("B", "cmd.exe /c echo B")
+                        {
+                            DependsOn = TaskDependencies.OnId("A"),
+                            ExitConditions = new ExitConditions()
+                            {
+                                // If task A exits with a non-zero exit code, run downstream task B.
+                                Default = new ExitOptions()
+                                {
+                                    DependencyAction = DependencyAction.Satisfy
+                                }
+                            }
+                        },
+
+                        // Task C is a parent task that cannot run and so causes a scheduling error.
+                        new CloudTask("C", "NonExistentProgram.exe"),
+                        // Task D depends on task C.
+                        new CloudTask("D", "cmd.exe /c echo D")
+                        {
+                            DependsOn = TaskDependencies.OnId("C"),
+                            ExitConditions = new ExitConditions()
+                            {
+                                // When task C exits with a scheduling error, the downstream task D will not run.
+                                SchedulingError = new ExitOptions()
```

Comment:
Yes, that makes sense. The example you looked at previously showed setting both default and scheduling error behavior under the same task (so you read it properly on your phone). I'll change the example to do that again.